core/capabilities/fakes/http_action.go

```diff
@@ -0,0 +1,204 @@
+package fakes
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/shopspring/decimal"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	commonCap "github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	customhttp "github.com/smartcontractkit/chainlink-common/pkg/capabilities/v2/actions/http"
+	httpserver "github.com/smartcontractkit/chainlink-common/pkg/capabilities/v2/actions/http/server"
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+	"github.com/smartcontractkit/chainlink-common/pkg/types/core"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+)
+
+var _ httpserver.ClientCapability = (*FakeHttpAction)(nil)
+var _ services.Service = (*FakeHttpAction)(nil)
+var _ commonCap.ExecutableCapability = (*FakeHttpAction)(nil)
+
+const HTTPActionID = "http-action@1.0.0"
+const HTTPActionServiceName = "HttpActionService"
+
+var fakeHttpActionInfo = capabilities.MustNewCapabilityInfo(
+	HTTPActionID,
+	capabilities.CapabilityTypeAction,
+	"An action that uses an HTTP request to run periodically at fixed times, dates, or intervals.",
+)
+
+type FakeHttpAction struct {
```

Comment:
It's not really fake. Maybe we should call it DirectHttpAction or something like that?